lib/primitives/array-element.js

```diff
@@ -38,6 +38,18 @@ module.exports = function(BaseElement, registry) {
       return [this.element, this.meta.toValue(), attributes, compactDoms];
     },
 
+    toEmbeddedRefract: function() {
+      if (this.attributes.content.length === 0 && this.meta.content.length === 0) {
```

Comment:
Seems like this could be abstracted into some helper method and make Demeter more happy.